### PR TITLE
Event list

### PR DESCRIPTION
**Pull request by taleymirza**
[1] Page should be accessible without logging in
[2] Page should be accessible at /events
[3] Pages will have a filters section on top, along with a search bar and event list as shown below
[4] Clicking on the event (card image or title) will open a blank page for now (later for details)
[5] Event should show, title, location, date, organisation etc

### CODE REVIEW

File: client/src/containers/ListEvents.js [hunk 1]
```diff
@@ -1,128 +1,133 @@
...
+    imageurl: 'https://cdn-images-1.medium.com/max/800/0*PkFv7ExBsCL5xrHA.',
+    organisation: 'GDG Kolachi',
+  },
+  {
+    key: '5',
+    title: 'Node School Workshop',
+    location: 'Malaysia',
+    startDateTime: '28-July-2018',
+    endDateTime: '29-July-2018',
+    imageurl: 'https://cdn-images-1.medium.com/max/800/0*PkFv7ExBsCL5xrHA.',
```
Comment (mashhoodr) [inline client/src/containers/ListEvents.js]: Please check the model in the documentation, I think this is named cover.

Comment (taleymirza) [reply client/src/containers/ListEvents.js]: @mashhoodr we need attribute for location in the model

Comment (taleymirza) [reply client/src/containers/ListEvents.js]: @mashhoodr , there is a suggestion for event model based on what we can have in event details page. we can further refine it.
Event Model:
- id
- title
- description
- cover
- starttime
- endtime
- ticketurl
- organization
	- name
	- logo
	- about
	- contact_details
- organizers
	- name
	- email
	- contact_num
- location
	- venue name
	- map location
- Sponsors

File: client/src/components/EventList/EventCard/EventCard.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  CardImg,
+  CardText,
+  CardBody,
+  CardTitle,
+  CardSubtitle,
+  Button,
+} from 'reactstrap';
+import { Link } from 'react-router-dom';
+
+export class EventCard extends Component {
```
Comment (mashhoodr) [inline client/src/components/EventList/EventCard/EventCard.js]: @QaseemLodhi @taleymirza please use the function syntax when no state is required

i.e. EventCard = (props) => { ...

File: client/src/containers/App/App.js [hunk 3]
```diff
@@ -20,6 +22,8 @@ class App extends Component<Props> {
         <Route path="/login" component={Login} />
         <Route path="/signup" component={Signup} />
         <Route path="/forgot-password" component={ForgotPassword} />
+        <Route path="/events" component={Events} />
+        <Route path="/details/:value" component={EventDetails} />
```
Comment (mashhoodr) [inline client/src/containers/App/App.js]: for event details we will have events/:event_id

File: client/src/containers/App/AuthContainer.js [hunk 4]
```diff
@@ -1,18 +1,14 @@
...
 import React, { Fragment } from 'react';
-import { EMNavbar } from '../../components/EMNavbar';
-import ListEvents from '../ListEvents';
+import Events from '../Events/Events';
 import { Container } from 'reactstrap';
 import { Route } from 'react-router-dom';
 
 const AuthContainer = () => (
-    <Fragment>
-        <EMNavbar />
```
Comment (mashhoodr) [inline client/src/containers/App/AuthContainer.js]: Why did we remove the Navbar from here?

File: client/src/containers/EventDetails/EventDetails.js [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+        {event ? (
+          <Container>
+            <CardImg top width="100%" src={event[0].cover} />
+            <Row className="block">
+              <Col className="text-header">
+                <h4>Event Summary</h4>
+              </Col>
+            </Row>
+            <Row className="block-content">
+              <SummaryContainer iconName="1" content={event[0].startDatetime} />
```
Comment (mashhoodr) [inline client/src/containers/EventDetails/EventDetails.js]: Shouldnt this be in a separate component?

**Comment by mashhoodr** [comment]
@QaseemLodhi please resolve conflicts

**Comment by mashhoodr** [comment]
@QaseemLodhi please fix all codacy issues

**Comment by mashhoodr** [comment]
@rahil051 please review this

File: client/src/containers/App/App.js (hunk 3 above)
Comment (QaseemLodhi) [reply client/src/containers/App/App.js]: was changed it but those changes are in another PR.

**Comment by taleymirza** [comment]
Done all fixes, merged with organisation branch

**Pull request closed without merging**